### PR TITLE
Support Joystream

### PR DESCRIPTION
**Pull request by Tbaut**
closes https://github.com/ChainSafe/Multix/issues/438

---

Submission checklist:

#### Layout

- [x] Change inspected in the desktop web ui
- [x] Change inspected in the mobile web ui

#### Compatibility

- [x] Functionality of change validated with a connected account with multisig
- [x] Applicable elements hidden / disabled for watched multisigs / pure
- [x] Looks good for solo multisig
- [x] Looks good for multisig with proxy

### CODE REVIEW

**Review by asnaith — CHANGES_REQUESTED** [review]
Looking good for the most part, I was able to create a multisig (without proxy as it says proxies are not supported on Joystream) and send a tx. 

However, there is an app crash if you try to set identity either from the options menu on the main page or the set identity dropdown on the "Send tx" modal. 

CleanShot 2023-11-15 at 15 27 48@2x

**Comment by Tbaut** [comment]
Great find. I guess they don't have the identity pallet as well, and Multix assumes it's there. I'll fix, and add a test.

**Comment by Tbaut** [comment]
Aaaand finally fixed the tests!

**Review by asnaith — APPROVED** [review]
Looks great, can no longer see the identity options...but that's not new information because your new test is already telling you that, awesome! 😎

🚀

File: packages/ui/src/utils/getAsMultiTx.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+interface Params {
+  api: ApiPromise
+  threshold: number
+  otherSignatories: string[]
+  tx?: SubmittableExtrinsic<'promise', ISubmittableResult> | HexString
+  weight?: Weight
+  when?: MultisigStorageInfo['when']
+}
+export const getAsMultiTx = ({ api, threshold, otherSignatories, tx, weight, when }: Params) => {
+  return api.tx.multisig.asMulti.meta.args.length === 6
```
Comment (Lykhoyda) [inline packages/ui/src/utils/getAsMultiTx.ts:16]: Can we make a "6" a constant that will describe itself? as it looks as magic number now :)

**Review by Lykhoyda — APPROVED** [review]
looks good! Please use some constant in getAsMultiTx file

File: packages/ui/src/utils/getAsMultiTx.ts (hunk 1 above)
Comment (Tbaut) [reply packages/ui/src/utils/getAsMultiTx.ts:16]: Good point, it is indeed a magic number as it's actually not future proof. Unfortunately we have no other choice. Joystream will migrate in the next months, and we can eventually remove this "if". I'll make this a constant such as `PARAM_LENGTH_OLD_ASMULTI`

**Pull request merged**